bin/fin

```diff
@@ -3811,9 +3811,11 @@ configure_mounts ()
 			fi
 
 			# Mount the share in VirtualBox
-			if ! is_docker_native; then
-				docker_machine_mount_nfs "${DOCKSAL_NFS_PATH}:${DOCKSAL_NFS_PATH}"
-			fi
+			# TODO: Disabled. To be removed.
+			# NFS mounts are now handled using NFS volumes - same approach as with Docker Desktop
+			#if ! is_docker_native; then
+			#	docker_machine_mount_nfs "${DOCKSAL_NFS_PATH}:${DOCKSAL_NFS_PATH}"
+			#fi
```

Comment:
I'm not 100% sure whether this should be dropped or kept. The reason for keeping would be to match the state between VBox and Docker Desktop. Docker Desktop has the paths like `/Users` shared inside the VM by default via `osxfs`. I'm wonder if we should just treat VBox the same way and keep the default `/User` NFS mount. @achekulaev thoughts?

Reply:
Leaving the default NFS share alone for now, since this would be a breaking change for some projects with custom/old stack configs. We also still have docs mentioning the old style usage of `$PROJECT_ROOT:/var/www` in volumes.